src/guides/v2.4/graphql/mutations/assign-customer-to-guest-cart.md

```diff
@@ -0,0 +1,108 @@
+---
+group: graphql
+title: assignCustomerToGuestCart mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `assignCustomerToGuestCart` mutation assigns a logged in customer and merge customer's shopping cart items to the specified guest shopping cart.
+
+The customer must be logged in (You must specify the customer’s authorization token in the headers).
+
+If this mutation is successful, the customer's shopping cart becomes inactive and all the products it contains are added to the contents of the guest cart (the active one).
+Customer now is assigned to that active cart.
+
+*Note:*
+Customer cart becomes inactive and the guest cart remains active; but for guest cart new `masked_id` is being generated.
```

Comment:
Hi @keharper , I mentioned them because I think it's a useful information for the developers, who can be confused why there is no old `masked_id` in the database, but a new one appeared instead.